bridge/checkpoint/checkpoint.go

```diff
@@ -126,8 +129,51 @@ func (c *checkpoint) determineCheckpointRange(lastChildBlock uint64) (start uint
 }
 
 func (c *checkpoint) generateCheckpoint(blocks []*types.Block) (*ctypes.Checkpoint, error) {
+	// Generate the Merkle root for the block hashes
+	blockHashes := make([][]byte, len(blocks))
+	for i, block := range blocks {
+		blockHashes[i] = block.Hash().Bytes()
```

Comment:
Block Hash is correct?

FYI, https://github.com/maticnetwork/bor/blob/d687d0d2de163e980c083f313504626cb1d3bdf3/consensus/bor/api.go#L161-L166

Reply:
Wait, so the Block Hash for the checkpoint is actually constructed from:
* The block number
* The generation timestamp
* The root of the txn trie
* The root of the receipts trie
?

Reply:
Please confirm that we should send this bundle for the Merkle root generation:
https://github.com/0xPolygon/polygon-edge/pull/484#discussion_r839655068

Reply:
Fixed 👍 

[4e758a8](https://github.com/0xPolygon/polygon-edge/pull/484/commits/4e758a8300d1d9c8423c12c292e89226d128c876)